src/events/crawler/scrapers/USA/NY/index.js

```diff
@@ -3,6 +3,7 @@ import * as parse from '../../../lib/parse.js';
 import * as transform from '../../../lib/transform.js';
 import * as datetime from '../../../lib/datetime.js';
 import * as geography from '../../../lib/geography.js';
+import getBoroughs from './borough.js';
```

Comment:
Does this need to be called `_borough.js`? I think the crawler will try to run this file in not. @qgolsteyn?